omero/sysadmins/unix/server-installation.txt

```diff
@@ -250,19 +250,18 @@ latest release is available from the :zeroc:`ZeroC website
 OMERO.web
 ^^^^^^^^^
 
-Please install one of :program:`apache` (and its wsgi module) or
-:program:`nginx` and Gunicorn in order to run OMERO.web.
+Please install :program:`nginx` and Gunicorn in order to run OMERO.web.
 
 +-----------+--------------------------------------+
 | System    | Packages                             |
 +===========+======================================+
-| BSD Ports | www/apache24 www/nginx               |
+| BSD Ports | www/nginx                            |
```

Comment:
this is the name of the package so yes we need it
I think the table should be kept